rd_ui/app/scripts/directives/plotly.js

```diff
@@ -121,6 +121,8 @@
   angular.module('plotly', [])
     .constant('ColorPalette', ColorPalette)
     .directive('plotlyChart', function () {
+      var baseHeight = 250;
```

Comment:
Why not calculate the height based on the height that is passed to the directive?